CI/before_script.msvc.sh

```diff
@@ -415,17 +413,24 @@ fi
 	if [ -z $APPVEYOR ]; then
 		cd $DEPS_INSTALL
 
+		# Boost's installer is still based on ms-dos API that doesn't support larger than 260 char path names
+		# We work around this by installing to root of the current working drive and then move it to our deps
+        
 		BOOST_SDK="$(real_pwd)/Boost"
+		CWD_DRIVE_ROOT=$(powershell -command '(get-location).Drive.Root')  # get the current working drive's root
```

Comment:
Nit; I'd love to see quotes around this too, even if there's no way that a drive root could contain a space today.

Reply:
Personally, I think quotes **and** running it through `sed` to turn the backslash into a forward slash is the best option.